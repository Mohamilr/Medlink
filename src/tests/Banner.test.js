import React from 'react';
import { render } from '@testing-library/react';
import Banner from '../components/banner/banner';

test('renders banner text', () => {
  const { getByText } = render(<Banner/>);
  const introText = getByText(/Bringing your nearest Healthcare facilities closer to you. Be it a Pharmacy or Hospital./i);
  expect(introText).toBeInTheDocument();
});
