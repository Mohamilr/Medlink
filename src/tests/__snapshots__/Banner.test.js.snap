// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders banner text 1`] = `
<DocumentFragment>
  <section>
    <div
      class="hero"
    >
      <div
        class="banner"
      >
        <img
          alt="Banner"
          src="https://res.cloudinary.com/project-s/image/upload/v1603972201/banner_eunugo.jpg"
        />
      </div>
      <div
        class="banner-text"
      >
        <div
          class="banner-text_intro"
        >
          <img
            alt="Hospital Cross"
            src="https://res.cloudinary.com/project-s/image/upload/v1603972199/medlink-cross_mpqhag.png"
          />
          <h3>
            Medlink
          </h3>
        </div>
        <p>
          Bringing your nearest Healthcare facilities closer to you. 
          <br />
          Be it a Pharmacy or Hospital.
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
